apps/meteor/definition/externals/meteor/accounts-base.d.ts

```diff
@@ -54,6 +54,14 @@ declare module 'meteor/accounts-base' {
 
 		const _accountData: Record<string, any>;
 
+		interface AccountsServerOptions {
+			ambiguousErrorMessages?: boolean;
+			restrictCreationByEmailDomain?: string | (() => string);
+			forbidClientAccountCreation?: boolean | undefined;
+		}
+
+		export const _options: AccountsServerOptions;
```

Comment:
export a private const?

Reply:
I didnt get your point. it starts with `_` but we use on our code, so its not private.
thats the reason I'm declaring this  prop actually to be used outside.